subsys/bluetooth/host/iso.c

```diff
@@ -982,89 +932,183 @@ static struct net_buf *hci_le_set_cig_params(struct bt_iso_create_param *param)
 	return rsp;
 }
 
-int bt_conn_bind_iso(struct bt_iso_create_param *param)
+static struct bt_iso_cig *get_free_cig(void)
 {
-	struct bt_conn *conn;
+	/* We can use the index in the `cigs` array as CIG ID */
+
+	for (int i = 0; i < ARRAY_SIZE(cigs); i++) {
+		if (!cigs[i].initialized) {
```

Comment:
Where is the initialized field set?

Reply:
Good catch! It never was, I guess I hadn't tried to create enough CIGs :)